src/test/regress/expected/create_drop_database_propagation.out

```diff
@@ -932,6 +928,19 @@ SELECT * FROM public.check_database_on_all_nodes('test_db') ORDER BY node_type;
 REVOKE CONNECT ON DATABASE test_db FROM propagated_role;
 DROP DATABASE test_db;
 DROP ROLE propagated_role, non_propagated_role;
+-- show that we don't try to propagate commands on non-distributed databases
+SET citus.enable_create_database_propagation TO OFF;
+CREATE DATABASE local_database_1;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+SET citus.enable_create_database_propagation TO ON;
+CREATE ROLE local_role_1;
+GRANT CONNECT, TEMPORARY, CREATE ON DATABASE local_database_1 TO local_role_1;
+ALTER DATABASE local_database_1 SET default_transaction_read_only = 'true';
+REVOKE CONNECT, TEMPORARY, CREATE ON DATABASE local_database_1 FROM local_role_1;
+DROP ROLE local_role_1;
+DROP DATABASE local_database_1;
```

Comment:
I think we still need the test for non-distributed database/role and if we send a command for non-distributed database/role, we need to give a meaningful error/warning
This note applies for all tests

Reply:
we already have such tests a few lines above?

```sql
CREATE DATABASE local_database_1;
....
GRANT CONNECT, TEMPORARY, CREATE ON DATABASE local_database_1 TO local_role_1;
ALTER DATABASE local_database_1 SET default_transaction_read_only = 'true';
```

Note that local_database_1 is a non-distributed database.

Reply:
Yes my bad. You're right. Thanks